tests/functional/add-mdss/container/vagrant_variables.yml

```diff
@@ -9,6 +9,7 @@ osd_vms: 1
 mds_vms: 1
 rgw_vms: 0
 nfs_vms: 0
+grafana_vms: 0
```

Comment:
I'm thinking, this should be `grafana_server_vms` if we really wanted  to be consistent ... :smiling_imp:

Reply:
+1